cmd/xl-v1-utils.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 // Returns number of errors that occurred the most (incl. nil) and the
-// corresponding error value. N B when there is more than one error value that
+// corresponding error value. NB When there is more than one error value that
```

Comment:
Then it should be N.B. -- it is a Latin abbreviation for 'Nota Bene' meaning "note well", or something along these lines of "watch out" / "be careful" ...

Of course we could also use PS which means "Post scriptum" which refers to an "afterthrough", or even PSS which is "Post scriptum scriptum" which is an "afterthrough" on an "afterthrough" but we are starting to digress now...    😉